### PR TITLE
Update about-admin-roles.md

### LINKED ISSUE #12820: Add search role details to common admin  role table
An external user also provided this feedback  - does it make sense to add Search  Admin role details to the common table ? 

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: f18d8345-78f3-0b16-f585-1e53f49d138f
* Version Independent ID: 015e98be-2db5-11a3-9abc-6f89b44c3f96
* Content: [About admin roles in the Microsoft 365 admin center - Microsoft 365 admin](https://learn.microsoft.com/en-us/microsoft-365/admin/add-users/about-admin-roles?view=o365-worldwide)
* Content Source: [microsoft-365/admin/add-users/about-admin-roles.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/add-users/about-admin-roles.md)
* Service: **microsoft-365-business**
* GitHub Login: @kwekuako
* Microsoft Alias: **kwekua**

### PR DESCRIPTION
**Pull request by msbemba**
Added Search admin to the list 

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12820

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/add-users/about-admin-roles.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ You'll probably only need to assign the following roles in your organization. By
 |Password admin  |   Assign the Password admin role to a user who needs to reset passwords for non-administrators and Password Administrators.   |
 |Power Platform admin |   Assign the Power Platform admin role to users who need to do the following: <br> - Manage all admin features for Power Apps, Power Automate, Power BI, Microsoft Fabric, and Microsoft Purview Data Loss Prevention <br> - Create and manage service requests <br> - Monitor service health  |
 |Reports reader |   Assign the Reports reader role to users who need to do the following: <br> - View usage data and the activity reports in the Microsoft 365 admin center <br> - Get access to the Power BI adoption content pack <br> - Get access to sign-in reports and activity in Azure AD <br> - View data returned by Microsoft Graph reporting API|
+|Search admin |  Assign the Search admin to users who need to create and manage search result content and define query settings for improved search results within the organization. Search admin manages the Microsoft Search configuration and can perform all of the content-management tasks a Search editor can.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/about-admin-roles.md]:
```suggestion
|Search admin |  Assign the Search admin role to users who need to create and manage search result content and define query settings for improved search results within the organization. The Search admin manages the Microsoft search configuration and can perform all the content-management tasks that a Search editor can.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**